src/vs/workbench/contrib/chat/browser/contrib/chatInputCompletions.ts

```diff
@@ -537,7 +541,7 @@ class BuiltinDynamicCompletions extends Disposable {
 				return;
 			}
 
-			const active = this.editorService.activeTextEditorControl;
+			const active = this.findActiveCodeEditor();
```

Comment:
Curious: in what case did `activeTextEditorControl` not return a good result that makes you go via  `codeEditorService.getActiveCodeEditor`? Is it for floating windows?